README.md

```diff
@@ -0,0 +1,84 @@
+# HashiCorp Vault Add-on for AWS EKS
+
+> Deploy HashiCorp Vault to AWS EKS Clusters with this add-on.
+
+## Table of Contents
+
+- [HashiCorp Vault Addon for AWS EKS](#hashicorp-vault-addon-for-aws-eks)
+  - [Table of Contents](#table-of-contents)
+  - [Usage](#usage)
+  - [Author Information](#author-information)
+  - [License](#license)
+
+## Usage
+
+If you would like to override any defaults with the chart, you can pass it via the `helm_config` variable.
+
+### Unsealing Vault
+
+Once the add-on has been deployed, the Vault server can be unsealed using the following commands.
+
+> You will need to be in the `vault` (Kubernetes) namespace while running these commands, by default
+
+You will first need to initialize the Vault server:
+
+```sh
+kubectl exec -it vault-0 -n vault -- vault operator init
+```
+
+Take note of the [unseal keys](https://www.vaultproject.io/docs/concepts/seal#seal-unseal) and [root token](https://www.vaultproject.io/docs/concepts/tokens#root-tokens) that get generated.
+
+Next, unseal the Vault server by providing at least _3_ of these keys to unseal Vault before servicing requests.
+
+```sh
+kubectl exec -it vault-0 -n vault -- vault operator unseal <key 1>
+kubectl exec -it vault-0 -n vault -- vault operator unseal <key 2>
+kubectl exec -it vault-0 -n vault -- vault operator unseal <key 3>
+ ```
+
+Confirm that the Vault server is unsealed by checking the status of the Vault server:
+
+```sh
+kubectl get pods -n vault | grep vault
+
```

Comment:
````suggestion
Confirm that the Vault server is unsealed by checking the status of the Vault server.

```sh
kubectl get pods -n vault | grep vault
```

This command will return a similar output to the following.

```sh

````

Reply:
this way, users can c/p the `kubectl` command by clicking the button

Reply:
Love it!